core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2Config.java

```diff
@@ -43,6 +43,7 @@
     private String authUrl;
     private String tokenUrl;
     private String jwksUrl;
+    private String userinfoUrl;
```

Comment:
Should we log a warning when this field is not set?

Reply:
This field is not required, if it's specified we use it to validate access tokens, otherwise we treat access tokens as JWTs.